backend/settings.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import logging.handlers
+import platform
+
+
+def init_logging():
+    logging_format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    log_filename = "/var/log/scrumsistant/scrum.log"
+    if platform.system() == "Windows":
+        # For us ghouls that are running on Windows, won't have a /var, so log locally
+        log_filename = "scrum.log"
+    # I would imagine that in production that we would just have a long running service that would log over
+    # a long period, so we might have a size determined rotation, but during development I find it helpful
+    # to restart the logs every run, seeing as different runs are common
```

Comment:
I'm not a fan of this decision - I might want to change a few things and see several runs; the startup log and/or using tail and [enter] is enough to help disambiguate. On your local you can clear the log files, on prod log rotation is a well understood external tool

Reply:
So no RotatingLogHandler at all, or just make it size based?

Reply:
Imo none at all but what do you think

Reply:
On CIQ prod we use 
https://linux.die.net/man/8/logrotate